packages/github-actions/actions/automerge-released-trunk/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Auto merge a released trunk
+description: Automatically merge trunk to develop after a release.
+
+runs:
+  using: composite
+  steps:
+    - name: "Merge the release to develop"
+      shell: bash
+      if: ${{ github.event.pull_request.merged == true && startsWith(github.head_ref, 'release/') && github.event.pull_request.user.login == 'github-actions[bot]' }}
+      # Use the github-actions bot account to commit.
+      # https://api.github.com/users/github-actions%5Bbot%5D
+      run: |
+        git config user.name github-actions[bot]
+        git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+        git fetch origin develop trunk --unshallow
+        git checkout develop
+        git pull
```

Comment:
As it just finished `git fetch`, and the `git checkout develop` operation can be done almost instantly, wonder if this line could be omitted?